Cargo.toml

```diff
@@ -226,7 +226,7 @@ ci = ["github"]
 installers = ["shell", "powershell"]
 # The archive format to use for windows builds (defaults .zip)
 windows-archive = ".zip"
-# The archive format to use for non-windows builds (defaults .tar.xz)
+# The archive format to use for non-windows builds (defaults .tar.gz)
```

Comment:
I think this actually _is_ the default, but we changed it for uv below.

Reply:
Gotcha, this was just really confusing because the comment doesn't match the implementation. Maybe best to clarify.